bitnami/pinniped/templates/concierge/service-api.yaml

```diff
@@ -0,0 +1,28 @@
+{{/* This service is meant to be consumed by the Kubernetes API, so we don't allow configuration on it */}}
+{{- if .Values.concierge.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "pinniped.concierge.api.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/part-of: pinniped
+    app.kubernetes.io/component: concierge
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations:
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  ports:
+    - protocol: TCP
+      port: 443
+      targetPort: http-api
```

Comment:
I think there is a typo here `https-api`

Reply:
Fixed